src/network/readme.md

```diff
@@ -0,0 +1,116 @@
+
+# Installation
+
+## Client
+### Linux
+Checkout the source specified above. Configure it with the following commands:
+`mkdir build && cd build && cmake .. -DBUILD_NETWORK_DEVICE=true -DBUILD_UNIT_TESTS=false -DBUILD_EXAMPLES=true -DBUILD_GRAPHICAL_EXAMPLES=true -DBUILD_WITH_TM2=false -DFORCE_RSUSB_BACKEND=true -DBUILD_SHARED_LIBS=true`
+
+After the build you can find the viewer, server and upgrade utility in *build/tools/realsense-viewer* and *build/tools/rs-server* folders.
+
+### Windows
+You can use the precompiled tools downloaded from the link specified in the deliverables section.
+To build from the source the following configuration command might be used:
+`cmake .. -G "Visual Studio 15 2017 Win64" -DCMAKE_BUILD_TYPE=RelWithDebInfo -DBUILD_NETWORK_DEVICE=true -DBUILD_UNIT_TESTS=false -DBUILD_EXAMPLES=true -DBUILD_GRAPHICAL_EXAMPLES=true -DBUILD_WITH_TM2=false -DBUILD_SHARED_LIBS=true`
+	
+## Server
+### Linux
+The same steps as building the client.
+
+### Windows
+No server supported now for on Windows.
+
+### RP4
+You can use the preconfigured image downloaded from the link specified in the deliverables section. Flash it into the microSD card using balenaEtcher and follow instruction from next section to configure it. If you have RP4 installed you might use the DEB package from the client tools package specified in the deliverables section.
+
+Also, you might want to built the server from the sources directly on RP4. To configure the source the following command might be used:
+`mkdir build && cd build && cmake .. -DBUILD_NETWORK_DEVICE=true -DBUILD_UNIT_TESTS=false -DBUILD_EXAMPLES=false -DBUILD_GRAPHICAL_EXAMPLES=false -DBUILD_WITH_TM2=false -DFORCE_RSUSB_BACKEND=true -DBUILD_SHARED_LIBS=true`
+
+After the build you can find the server and upgrade utility in *build/tools/rs-server* folder.
+
+TODO: cross-compilation explanation.
+
+# Board access.
+
+There are two scenarios to access the board. One assumes the user can connect monitor and keyboard to the board. The second scenario allows to access the board using USB connection only.
+
+You will need Bonjour as explained [here](https://learn.adafruit.com/bonjour-zeroconf-networking-for-windows-and-linux/). Most Linux systems already have it installed. 
+
+## Linux
+1. After connecting to USB make the new wired connection shared using nm-connection-editor (install it if needed).
+1. Make SSH connection to pi@raspberrypi.local.
+1. Login using *pi* username and *raspberry* for the password.
+1. Adjust */etc/dhcpcd.conf* as whitepaper explains.
+1. Start *realsense-viewer* on the other side.
+
+## Windows
+1. Use Putty to open SSH connection to pi@raspberrypi.local.
+1. Login using *pi* username and *raspberry* for the password.
+1. Adjust */etc/dhcpcd.conf* as whitepaper explains.
+1. Start realsense-viewer on the other side.
+[Source](https://learn.adafruit.com/turning-your-raspberry-pi-zero-into-a-usb-gadget/ethernet-gadget)
```

Comment:
Not sure what the source link is related? it is on "Start realsense-viewer on the other side." line
What does source mean?